internal/host/hostcommands/download_boot_artifacts_cmd_test.go

```diff
@@ -0,0 +1,88 @@
+package hostcommands
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/host/hostutil"
+	"github.com/openshift/assisted-service/internal/versions"
+	"github.com/openshift/assisted-service/models"
+	"github.com/openshift/assisted-service/pkg/auth"
+	"gorm.io/gorm"
+)
+
+var _ = Describe("download_boot_artifacts_cmd", func() {
```

Comment:
You're testing `GetSteps` so that's what should be in the `Describe` here.

Maybe `downloadBootArtifactsCmd.GetSteps` if you want to be explicit.

Reply:
Modified the description, thanks!